packages/react-ui-components/src/Icon/index.js

```diff
@@ -1,16 +1,22 @@
-import './fontAwesome/font-face.css';
-import icons from './fontAwesome/icons.css';
+import fontawesome from '@fortawesome/fontawesome';
+import brands from '@fortawesome/fontawesome-free-brands';
+import solid from '@fortawesome/fontawesome-free-solid';
+import regular from '@fortawesome/fontawesome-free-regular';
 import {themr} from 'react-css-themr';
 import identifiers from './../identifiers.js';
 import injectProps from './../_lib/injectProps.js';
 import {makeValidateId, makeGetClassName} from './../_lib/fontAwesome.js';
 import style from './style.css';
 import Icon from './icon.js';
 
+fontawesome.library.add(brands, solid, regular);
+
 const ThemedIcon = themr(identifiers.icon, style)(Icon);
 
 export default injectProps({
-    iconMap: icons,
+    //TODO: What to do about the icon map?
```

Comment:
Can't we just remove it?

Reply:
I hope so. I'm not a 100% sure what it did with the old icons, and if we still need it somehow. So we kept it and wanted to leave the decision to remove it to somebody who knows more about the new Ui.